RecoJets/JetProducers/python/ak8PFJets_cfi.py

```diff
@@ -16,7 +16,8 @@
 # AK8 jets with various pileup subtraction schemes
 ##############################################################################
 ak8PFJetsPuppi = ak8PFJets.clone(
-    src = cms.InputTag("puppi")
+    src = cms.InputTag("particleFlow"),
+    applyPuppiWeight = cms.bool(True)
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.